bin/ubuntu-core-initramfs

```diff
@@ -360,7 +360,6 @@ def create_efi(parser, args):
         parser.error("--stub is required, and one was not automatically detected")
     if args.root:
         args.stub = path_join_make_rel_paths(args.root, args.stub)
-        args.sbat = path_join_make_rel_paths(args.root, args.sbat)
```

Comment:
this is security sensitive for our signing policy, i don't believe we can just drop sbat, as no kernel.efi will be bootable then.

why is sbat.txt support being removed? or how is it replaced?

Reply:
hm, in mantic i do see that:

# objdump -h /usr/lib/systemd/boot/efi/linuxx64.efi.stub | grep sbat
  6 .sbat         000000e0  000000000001a000  000000000001a000  00011000  2**2

sbat section is present in the stub.  maybe it's good enough.

Reply:
sbat in the pre-compiled stub is fine. FTR, the current content is
```
sbat,1,SBAT Version,sbat,1,https://github.com/rhboot/shim/blob/main/SBAT.md
systemd,1,The systemd Developers,systemd,252,https://systemd.io/
systemd.ubuntu,1,Ubuntu,systemd,252.5-2ubuntu3,https://bugs.launchpad.net/ubuntu/
```
which is an updated version of what we had in the repo:
```
sbat,1,SBAT Version,sbat,1,https://github.com/rhboot/shim/blob/main/SBAT.md
systemd,1,The systemd Developers,systemd,245,https://www.freedesktop.org/wiki/Software/systemd
systemd.ubuntu,1,Ubuntu,systemd,245.4-4ubuntu3.6,https://bugs.launchpad.net/ubuntu/
```